go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/linode/packer-plugin-linode
 go 1.19
 
 require (
-	github.com/go-resty/resty/v2 v2.3.0 // indirect
+	github.com/go-resty/resty/v2 v2.7.0 // indirect
```

Comment:
Can we move indirect dependency to the below section? I think this can be done by removing this line then `go mod tidy`.